ozaria/site/styles/common/common.sass

```diff
@@ -1,11 +1,26 @@
 @import "ozaria/site/styles/common/variables"
 
+.ozaria-button
+  width: 182px
+  height: 45px
+  box-sizing: border-box
+  
 .ozaria-primary-button
-  background-image: url('/images/ozaria/level/primary_button.png')
+  background-color: #1fbab4
   padding: 11px 19px
   color: #f7f9f9
   font-family: $body-font-style
   font-size: 18px
   font-weight: bold
   letter-spacing: 0.71px
-  line-height: 24px
\ No newline at end of file
+  line-height: 24px
+
+.ozaria-secondary-button
+  padding: 11px 19px
+  color: #1fbab4
+  font-family: "Open Sans"
```

Comment:
Is this font family stored in a variable?

Reply:
Yes, will update